src/App.jsx

```diff
@@ -70,6 +71,7 @@ const App = () => {
           <Route path="search" element={<Search />} />
           <Route path="search/:key" element={<Search />} />
           <Route path="user/:id" element={<PublicProfile />} />
+          <Route path='set/:id/newCard' element={<CreateCard/>}/>
```

Comment:
Why do we have the id here? 
Isn't the idea already existed in the components/sets/Sets ?